packages/marketplace/src/styles/pages/developer-settings-organisation-tab.scss

```diff
@@ -0,0 +1,46 @@
+.nameColumn {
```

Comment:
Can you remove this file please and use elements. All of this CSS exists as a util, a component or a Bulma class

Reply:
This comment is still outstanding